pmd-cataloger/src/test/java/sfdc/sfdx/scanner/pmd/MainMessagesTest.java

```diff
@@ -17,6 +18,7 @@
 
 public class MainMessagesTest {
 
+	@Before
```

Comment:
This was failing when multiple tests were run and another test ran before this which added to the SfdxMessager.